src/components/LoginButton.vue

```diff
@@ -31,7 +30,7 @@ export default {
 };
 </script>
 <style lang="scss">
-$btn-size: 3.3rem;
+@import "@/assets/style.scss";
```

Comment:
@JatinThakur14 rename this to variables.scss - it isnt a style file

Reply:
okay